src/lib/template/dom-repeat.html

```diff
@@ -370,7 +370,7 @@
       // Apply user filter to keys
       if (this._filterFn) {
         this._orderedKeys = this._orderedKeys.filter(function(a) {
-          return this._filterFn(c.getItem(a));
```

Comment:
to be totally inline with the filter api we should also pass the original array as well, thoughts?